service/handlers.go

```diff
@@ -604,14 +604,25 @@ func (h handler) getProject(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	level.Debug(l).Log("message", "getting project")
+	level.Debug(l).Log("message", "getting project from credentials provider")
 	resp, err := cp.GetProject(projectName)
```

Comment:
Since the DB is called afterwards, should we just ditch calling the creds provider and just call the DB? I know the creds provider is the "source of truth", but the calls seem repetitive.